opendbc/car/ford/values.py

```diff
@@ -123,41 +134,41 @@ def init(self):
 
 class CAR(Platforms):
   FORD_BRONCO_SPORT_MK1 = FordPlatformConfig(
-    [FordCarDocs("Ford Bronco Sport 2021-24")],
+    [FordCarDocs("Ford Bronco Sport 2021-24", footnotes=[Footnote.HW_TYPE])],
```

Comment:
If you want a footnote to show up on all cars you should be able to set a default for `footnotes` in FordCarDocs instead